extensions/wikia/SpecialHealthcheck/HealthController.class.php

```diff
@@ -108,20 +108,19 @@ private function occursInArray( $needle, $array ) {
 
 	/**
 	 * Execute checks for a single database server
-	 * @param string $databaseName Database name to use for connection
 	 * @param LoadBalancer $loadBalancer Load Balancer instance for the given cluster
 	 * @param int $index Server index to test
 	 * @return bool Is server healthy?
 	 * @throws MWException
 	 */
-	private function testHost( $databaseName, LoadBalancer $loadBalancer, $index ) {
+	private function testHost(LoadBalancer $loadBalancer, $index ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 5, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)